src/core/include/units/quantity_cast.h

```diff
@@ -78,18 +75,17 @@ template<typename From, typename To>
 struct cast_traits;
 
 template<typename From, typename To>
-  requires common_type_with_<std::common_type_t<From, To>, std::intmax_t>
-struct cast_traits<From, To> {
+  requires common_type_with_<std::common_type_t<From, To>,
+                             std::intmax_t> struct cast_traits<From, To> {
   using ratio_type = std::common_type_t<std::common_type_t<From, To>, std::intmax_t>;
   using rep_type = ratio_type;
 };
 
 template<typename From, typename To>
-  requires (!common_type_with_<std::common_type_t<From, To>, std::intmax_t>) &&
-          scalable_number_<std::common_type_t<From, To>, std::intmax_t> &&
-          requires { typename std::common_type_t<From, To>::value_type; } &&
-          common_type_with_<typename std::common_type_t<From, To>::value_type, std::intmax_t>
-struct cast_traits<From, To> {
+  requires(!common_type_with_<std::common_type_t<From, To>, std::intmax_t>) &&
+          scalable_number_<std::common_type_t<From, To>,
+                           std::intmax_t>&& requires { typename std::common_type_t<From, To>::value_type; } &&
+  common_type_with_<typename std::common_type_t<From, To>::value_type, std::intmax_t> struct cast_traits<From, To> {
```

Comment:
```suggestion
  requires(!common_type_with_<std::common_type_t<From, To>, std::intmax_t> &&
          scalable_number_<std::common_type_t<From, To>,
                           std::intmax_t>&& requires { typename std::common_type_t<From, To>::value_type; } &&
  common_type_with_<typename std::common_type_t<From, To>::value_type, std::intmax_t>) struct cast_traits<From, To> {
```
This should result in better formatting.